ruby_on_rails/introduction/web_refresher.md

```diff
@@ -2,23 +2,27 @@
 
 To really understand how Rails works, you need to have a solid base in the guts of the web.  You've already covered some of this in previous sections (and if you went through the Ruby curriculum you can just skim it), but this time you'll get a chance (in the project) to reach out and make some real web requests.
 
-### Learning Outcomes
-By the end of this lesson, you should be able to do the following:
-
-* Explain the basics of HTTP.
-* Identify the 4 most commonly used HTTP verbs.
-* Identify the 7 RESTful routes of Rails.
-* Name the different components of a URL.
-* Explain the basics of MVC.
-* Describe what an API is.
-* Explain what "cookies" and "sessions" are.
-* Describe the difference between "authentication" and "authorization".
+### Lesson Overview
+This section contains a general overview of topics that you will learn in this lesson.
+
+* The basics of HTTP.
+* The 4 most commonly used HTTP verbs.
+* The 7 RESTful routes of Rails.
+* The different components of a URL.
+* The basics of MVC.
+* What an API is.
+* What "cookies" and "sessions" are.
+* The difference between "authentication" and "authorization".
```

Comment:
Two minor nits, if my read on the style guide is correct we want to use hyphens for lists as well as two spaces between list items and content

Reply:
Ah, forgot about hyphens here. Now as for the spaces... it says 2 spaces for numbered lists and 3 spaces for unordered ones and 4 spaces for nested lists. Honestly it flew by me because I wasn't really focused on nested list formatting, was more concerned with it rendering properly. Fixing now.